packages/nodejs/src/__tests__/client.test.ts

```diff
@@ -12,23 +12,35 @@ describe("BaseClient", () => {
 
   let client: BaseClient
 
-  // enableMinutelyProbes is set to false so we don't leak timers
-  const DEFAULT_OPTS = { name, pushApiKey, enableMinutelyProbes: false }
+  const DEFAULT_OPTS = { name, pushApiKey }
 
   beforeEach(() => {
     client = new BaseClient({ ...DEFAULT_OPTS })
   })
 
+  afterEach(() => {
+    client.stop()
+  })
+
   it("starts the client", () => {
     const startSpy = jest.spyOn(client.extension, "start")
     client.start()
     expect(startSpy).toHaveBeenCalled()
   })
 
   it("stops the client", () => {
-    const stopSpy = jest.spyOn(client.extension, "stop")
+    const extensionStopSpy = jest.spyOn(client.extension, "stop")
```

Comment:
```suggestion
    const extensionStopSpy = jest.spyOn(Extension.prototype, "stop")
```

To keep it consistent with the changes from https://github.com/appsignal/appsignal-nodejs/pull/613/files#diff-1f789c834ce7476a0d619f0b2c0c672ffa5321582393e392b25204ad30dc51b1R47